src/pages/new-claim/new-claim.tsx

```diff
@@ -21,18 +21,23 @@ interface FormState {
   amount: string;
 }
 
-function getValidationMessages(form: FormState) {
+function getValidationMessages(form: FormState, policy: Policy) {
   const messages: ValidationMessages<FormState> = {};
 
   if (form.evidence.trim().length === 0) {
     messages.evidence = 'Please fill in this field';
   }
 
-  const result = goSync(() => parseApi3(form.amount));
+  const result = goSync(() => parseClaimAmount(form.amount));
```

Comment:
I see that the `parseClaimAmount` function also trims the value. Can you check if we should not do the same in the rest of the dashboard? (I think we handle validation quite well there, but just to make sure)

Reply:
Did you look at it? Do we handle the validation in the rest of the code correctly?

Reply:
I just dug a bit into this, and it seems we are missing some trimming for the New Proposal form and the Delegate form. E.g. valid addresses padded with spaces shows an "invalid address" validation message. I will create an issue for this 👍